stream-chat-android-client/src/main/java/io/getstream/chat/android/client/ChatClient.kt

```diff
@@ -704,11 +706,12 @@ public class ChatClient internal constructor(
                         cid = cid,
                         reaction = reaction,
                         enforceUnique = enforceUnique,
-                        currentUser = currentUser,
+                        currentUser = currentUser!!,
```

Comment:
maybe requireNotNull instead of `!!`

Reply:
I think we can leave `!!` in this case as it's guaranteed the user won't be null here. We have a check for that in `precondition`